bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
Can you link me to a file where we import Component without React? I didnt know that was possible and i cant find where we do it?